coretex/cli/resources/_coretex.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
```

Comment:
Extract _coretex.py script from node into coretexpylib resources and use the same script in node and in cli